README.md

```diff
@@ -13,4 +13,10 @@ $ make test-down  # delete containers + volumes
 In case of unit tests (you do not need db):
 ```
 $ pytest test/unit/test_operator.py
-```
\ No newline at end of file
+```
+
+# context
```

Comment:
What does `context` represent in this context (pun intended :) ) ?
Also, it would be awesome if we could expand the readme a bit more, with how to use etc

Reply:
it was more like a note for me. its what contessa gives you to use in sql beside what you pass.

ad readme - gonna write once there data-quality operator would be finished